promql/parse.go

```diff
@@ -112,6 +120,105 @@ func (p *parser) parseExpr() (expr Expr, err error) {
 	return
 }
 
+// sequenceValue is a omittable value in a sequence of time series values.
+type sequenceValue struct {
+	Value   clientmodel.SampleValue
```

Comment:
Even for internal types I find it a good way to signal that a value is free for modification outside of the types' own methods.